pynn_spinnaker/simulator.py

```diff
@@ -340,6 +351,9 @@ def _load_current_input_verts(self, placements, allocations,
                 # Select placed chip
                 with self.machine_controller(x=vertex_placement[0],
                                              y=vertex_placement[1]):
+                    # Configure watchdog for this vertex
+                    self._configure_software_watchdog()
```

Comment:
``set(itervalues(placements))`` would certainly be efficient (and save you prodding chips you don't use). That said I often just iterate over a SystemInfo object which gives (x, y) pairs of all working chips...